src/grandorgue/settings/GOSettingsPorts.h

```diff
@@ -20,20 +20,53 @@ class GOSettingsPorts
 {
 private:
   const GOPortFactory& m_PortFactory;
+
+  struct PortItemData: public wxClientData
+  {
+    wxString m_PortName;
+    wxString m_ApiName;
+
+    PortItemData(const wxString& portName, const wxString &apiName = wxEmptyString)
+    {
+      m_PortName = portName;
+      m_ApiName = apiName;
+    }
+
+    bool isFor(const wxString& portName, const wxString &apiName = wxEmptyString)
```

Comment:
isFor is a bit cryptic, I think. If it could it be specified more clearly what this "is for", it would be great.

Reply:
Renamed to isItemForPortApi